raw_data/preprocessed/africa/kenya_english.txt

```diff
@@ -22,9 +24,9 @@ PART III - ADMINISTRATION The National Environment Council
ent times. [4] The members appointed under paragraphs [a], [c] and [e] of subsection [1] shall hold office for a term of four years and shall be eligible for reappointment for one further term of four years. [5] The Board shall elect a vice-chairman from among the members appointed under paragraph [e] of subsection [1]. [6] The Board shall meet at least four times in every financial year. [7] The Chairman shall preside at every meeting of the Board at which he is present, but in his absence the vice-chairman shall preside, and in his absence, the members present shall elect one of their member who shall, with respect to that meeting and the business transacted thereat have all the powers of the Chairman. [8] Unless a unanimous decision is reached, a decision on any matter before the Board shall be by a majority of votes of the members present and in the case of an equality of votes, the Chairman or person presiding shall have a casting vote. [9] The quorum for the transaction of the business of the Board shall be seven members including the person presiding; and all acts, matters or things authorized or required to be done by the Board, shall be effected by a resolution passed by a majority of the members present and voting. [10] The Secretary to the Board shall not be entitled to vote on any matter before the Board. [11] Appointment of a member of the Board may be terminated by the appointing authority where the member: [a] is adjudged bankrupt or enters into a composition or scheme of arrangement with his creditors; [b] is convicted of an offence and sentenced to imprisonment for a term of six months or more without the option of a fine; [c] is incapacitated by prolonged physical or mental illness from performing his duties as a member of the Board; or [d] becomes, for any reason, incapable or incompetent of properly performing the functions of his office. [12] Where a member of the Board dies or resigns or otherwise vacates office before the expiry of his term of office, the appointing authority shall appoint another person in the place of such member. [13] Where the Director-General is unable to perform the functions of his office due to any temporary incapacity which is likely to be prolonged, the President may appoint a substitute therefore to act with the full powers of the Director-General until such time as the President determines that the incapacity has ceased. [14] [a] The Director-General shall be the chief executive of the Authority and shall, subject to this Act, be responsible for the day to day management of the affairs of the Authority. [b] A Director shall perform such functions as are conferred by this Act and such additional duties as may be assigned by the Director-General. [c] The Director-General and the Directors of the Authority shall be paid such salaries and allowances as may, from time to time, be determined by the President. [15] Subject to subsections [6], [7], [8] and [9] the Board shall regulate its own Procedure.
```

Comment:
`section 1[a]` should have been replaced with `subsections [1][a]`, not `subsections 1[a]`.